src/components/Docs/SnippetViewer/BatchCheckRequestViewer.tsx

```diff
@@ -275,60 +275,107 @@ response = await fga_client.batch_check(ClientBatchCheckRequest(checks=checks),
         .join(', ')}]`;
 
     case SupportedLanguage.JAVA_SDK:
-      return `// The Java SDK does not yet support server-side batch checks. This currently just calls the check endpoint in parallel.
-
-var request = List.of(
-    ${checks
-      .map(
-        (check) => `new ClientCheckRequest()
-        .user("${check.user}")
-        .relation("${check.relation}")
-        ._object("${check.object}")${
-          check.contextualTuples
-            ? `\n        .contextualTuples(List.of(
-            ${check.contextualTuples
-              .map(
-                (tuple) => `new ClientTupleKey()
-                .user("${tuple.user}")
-                .relation("${tuple.relation}")
-                ._object("${tuple.object}")`,
-              )
-              .join(',\n            ')})
-        )`
-            : ''
-        }${check.context ? `.context(${check.context})` : ''}`,
-      )
-      .join(',\n    ')}
+      return ` // Requires >=v0.8.0 for the server side BatchCheck, earlier versions support a client-side BatchCheck with a slightly different interface
+var reequst = new ClientBatchCheckRequest().checks(
+    List.of(
+        new ClientBatchCheckItem()
+            .user("user:81684243-9356-4421-8fbf-a4f8d36aa31b")
```

Comment:
These IDs are hardcoded, instead you want to build it dynamically based on what was passed in, similar to how it previously was (e.g. based on the AuthZ Model, the users, contextual tuples, expected response, etc..)

Reply:
Updated to not hard-code